graphql/util.go

```diff
@@ -764,6 +752,31 @@ func ModifyVersionHandler(ctx context.Context, dataConnector data.Connector, pat
 	if err != nil {
 		return mapHTTPStatusToGqlError(ctx, httpStatus, err)
 	}
+
+	if version.Requester == evergreen.PatchVersionRequester || version.Requester == evergreen.GithubPRRequester {
+		// restart is handled through graphql because we need the user to specify
+		// which downstream tasks they want to restart
+		if modifications.Action != Restart {
+			//do the same for child patches
+			p, err := patch.FindOneId(patchID)
+			if err != nil {
+				return ResourceNotFound.Send(ctx, fmt.Sprintf("error finding patch %s: %s", patchID, err.Error()))
+			}
+			if p == nil {
+				return ResourceNotFound.Send(ctx, fmt.Sprintf("patch not found %s: %s", patchID, err.Error()))
```

Comment:
This shouldn't have err.Error() since it's nil

Reply:
can this be "patch '%s' not found" instead?

Reply:
done